packages/map-template/src/components/MapsIndoorsMap/MapsIndoorsMap.jsx

```diff
@@ -30,8 +30,10 @@ let _locationsDisabled;
  * @param {string} [props.primaryColor] - If you want the splash screen to have a custom primary color, provide the value here.
  * @param {string} [props.logo] - If you want the splash screen to have a custom logo, provide the image path or address here.
  * @param {array} [props.appUserRoles] - If you want the map to behave differently for specific users, set one or more app user roles here.
+ * @param {array} [props.externalIds] - Filter locations shown on the map based on the external ids.
```

Comment:
Remember to update the readme to reflect all the props.

Also, should it be `externalIDs`? 🤔

Reply:
✅